patch/files/scripts/outside.gd

```diff
@@ -937,19 +986,26 @@ func _on_purchasebutton_pressed():
 	slaveguildslaves(guildlocation)
 	clearselection('buy')
 
-
-
 func _on_slavesellbutton_pressed():
 	var upgradefromslave = false
 	var text = ''
+	var racepricemodchange = 0 #ralph5
+	var racepricemod = 1 #ralph5
 	globals.resources.gold += selectedslaveprice
-	if selectedslave.obed >= 80 && selectedslave.fromguild == false && selectedslave.effects.has('captured') == false:
+	#ralph5
+	racepricemod = globals.state.racemarketsat[selectedslave.race.replace('Halfkin', 'Beastkin')] #ralph5
+	racepricemodchange = (racepricemod - 0.5)*0.1 #the bigger the premium the more the premium will be decreased
+	#print("Pricemod for " + selectedslave.race + " is " + str(racepricemod))
+	racepricemod = clamp(racepricemod - racepricemodchange,0.5,5)
+	globals.state.racemarketsat[selectedslave.race] = racepricemod
+	#print("Sold 1 " + selectedslave.race + "   Pricemod decreased " + str(racepricemodchange) + " to " + str(racepricemod))
+	#/ralph5
+	if selectedslave.obed >= 90 && selectedslave.fromguild == false && selectedslave.effects.has('captured') == false:
 		upgradefromslave = true
-		globals.resources.upgradepoints += globals.originsarray.find(selectedslave.origins)+1
-		###---Added by Expansion---### Person Expanded / Breeder Specialty
+		###---Added by Expansion---###
 		globals.expansion.updatePerson(selectedslave)
 		if selectedslave.npcexpanded.mansionbred == true && globals.state.spec == 'Breeder':
-			globals.resources.upgradepoints += (globals.originsarray.find(selectedslave.origins)+1)*2
+			globals.resources.upgradepoints += (globals.originsarray.find(selectedslave.origins)+1)*1.5 #ralph3

```

Comment:
A change from *2 to *1.5 without condition

Reply:
Fixed